src/main/java/com/rakovets/course/java/core/practice/concurrency/master_and_worker/Master.java

```diff
@@ -0,0 +1,26 @@
+package com.rakovets.course.java.core.practice.concurrency.master_and_worker;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class Master extends Thread{
```

Comment:
"Violation: There isn't separating white spaces for curly brake
Solution: Add white spaces"